scripts/create_newcase

```diff
@@ -1060,6 +1066,53 @@ if( -e "$machdir/Depends.$compiler" ) {
     $sysmod = "cp $machdir/Depends.$compiler $caseroot/";
     system($sysmod) == 0 or die "ERROR: $sysmod failed: $?\n";
 }
+}
+
+# The following creates cesm_postbuild, cesm_prerun, cesm_postrun, cesm_postrun2, and mach_syslog
+# The following creates cesm_postbuild
+if( -e "$scriptsroot/Tools/cesm_postbuild" ) {
+  $sysmod = "cp $scriptsroot/Tools/cesm_postbuild $caseroot/Tools/cesm_postbuild";
+  system($sysmod) == 0 or die "ERROR: $sysmod failed: $?\n";
+}else{
+  $sysmod = "cp $scriptsroot/Tools/cesm_postbuild.noop $caseroot/Tools/cesm_postbuildn";
```

Comment:
Yes. This is a typo, and it came from me (is in my version also). This is an if-test branch that is not currently being executed, and that I added as an example of what we might want to do if releasing this without the performance archiving logic.

Should be

  $sysmod = "cp $scriptsroot/Tools/cesm_postbuild.noop $caseroot/Tools/cesm_postbuild";